README.md

```diff
@@ -21,78 +21,100 @@ the Arelle plugin.  The preparation process updates the iXBRL file to include:
 
 Once prepared, the resulting file provides an entirely standalone viewer.  Once
 prepared, the viewer is entirely standalone, and does not require access to the
-taxonomy, or to any online services.  The only dependency is on the Javascript
-viewer application, which is a single file which may be stored locally.
+taxonomy, or to any online services.  The only dependency is on the javascript
+viewer application, which is a single file which can be accessed directly online, downloaded or built locally.
 
-## Building the ixbrlviewer
+The javascript viewer application is a single Javascript file called ixbrlviewer.js. It
+contains all of the javascript that runs the viewer functionality.
 
-The viewer works using a single Javascript file called ixbrlviewer.js. It
-contains all of the javascript that runs the viewer functionality. In order to
-successfully build an ixbrl-viewer you need to first build the ixbrlviewer
-file.
+## Installation
 
 1. Clone the [iXBRL Viewer git repository][ixbrlviewer-github].
-2. Install npm. Instructions can be found here: https://www.npmjs.com/get-npm
-3. Install the dependencies for javascript by running: `npm install`.  This
+2. Download and install [Arelle][arelle-download]
+
+
+## Accessing the javascript viewer application
+
+### Accessing via the CDN
+In order to make things as easy as possible Workiva is now hosting the javascript 
+via a CDN. It can be accessed via the followng CDN url: 
+```
+https://cdn-prod.wdesk.org/ixbrl-viewer/<tagged version>/ixbrlviewer.js
+```
+Where `<tagged version>` is the current version of ixbrl-viewer you are using. For instance `0.1.58`.
```

Comment:
The link dosnt work, because the next release is technically `0.1.58`. So it wont work until I release, which I was planning on doing right after this merges.